newIDE/app/src/CompactPropertiesEditor/index.js

```diff
@@ -998,6 +1009,9 @@ const CompactPropertiesEditor = ({
         return contentView;
       } else if (field.valueType === 'resource') {
         return renderResourceField(field);
+      } else if (field.valueType === 'leaderboardId') {
+        // TODO Make a compact version of LeaderboardIdPropertyField
+        return null;
```

Comment:
I think it needs to be designed first.
The property won't be shown in the side panel. I guess, it's not really an issue.